resources/repository.rb

```diff
@@ -0,0 +1,88 @@
+# frozen_string_literal: true
+#
+# Cookbook:: postgresql
+# Resource:: repository
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+property :version, String, default: '10'
+property :minor_version, String, default: '10-1'
+property :enable_pgdg, [true,false], default: false
```

Comment:
pretty sure this was meant to default to true right?

Reply:
I did in the end I'll push this up in a bit